.github/workflows/wheels-dependencies.sh

```diff
@@ -105,7 +105,7 @@ function build_libavif {
     python3 -m pip install meson ninja
 
     if [[ "$PLAT" == "x86_64" ]] || [ -n "$SANITIZER" ]; then
-        build_simple nasm 2.16.03 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03
```

Comment:
Fyi, if the nasm.us outage is temporary, then the presence of https://github.com/python-pillow/pillow-depends/blob/main/nasm-2.16.03.tar.gz should allow this to continue working until it comes back.

Reply:
I'm not sure it's temporary, since it's been down now for nearly a week. But I had forgotten about the pillow-depends backup. I've reverted this change.

Reply:
I looked into that. The source tarballs on nasm.us have the autoconf and automake artifacts included. So we couldn't just swap out with a GitHub tag archive here, as we would need to update this script to call autogen.sh. Not an insurmountable obstacle, but it couldn't use the `build_simple` bash function from multibuild.

Reply:
nasm.us is back now.